starkbank/dictkey/__dictkey.py

```diff
@@ -17,18 +17,15 @@ class DictKey(Resource):
     - owner_type [string]: DICT key owner type. ex "naturalPerson" or "legalPerson"
     - bank_name [string]: bank name associated with the DICT key. ex: "Stark Bank"
     - ispb [string]: bank ISPB associated with the DICT key. ex: "20018183"
-    - branch_code [string]: bank account branch code associated with the DICT key. ex: "9585"
-    - account_number [string]: bank account number associated with the DICT key. ex: "9828282578010513"
+    - branch_code [string]: bank account branch code associated with the DICT key. ex: "ZW5jcnlwdGVkLWJyYW5jaC1jb2Rl"
```

Comment:
bank account encrypted branch code associated with the DICT key. ex: "ZW5jcnlwdGVkLWJyYW5jaC1jb2Rl"